nucliadb_utils/nucliadb_utils/audit/stream.py

```diff
@@ -99,9 +113,17 @@ async def initialize(self):
 
         self.js = get_traced_jetstream(self.nc, self.service)
         self.task = asyncio.create_task(self.run())
+
+        self.kb_usage_utility = KbUsageReportUtility(
```

Comment:
Feels weird to have a utility inside another one -- utilities are meant to be singletons in nucliadb.
Proposal: why not calling it `KbUsageReportClient` instead?

Reply:
no strong opinion though, feel free to disregard the comment !